src/plugins/intel_cpu/tests/unit/CMakeLists.txt

```diff
@@ -14,14 +14,14 @@ endif()
 
 if(NOT (ARM OR AARCH64))
     list(APPEND EXCLUDED_SOURCE_PATHS_FOR_UNIT_TEST
-      ${CMAKE_CURRENT_SOURCE_DIR}/ngraph_transformations/arm)
+      ${CMAKE_CURRENT_SOURCE_DIR}/ov_transformations/arm)
 endif()
 
 if(NOT X86_64)
     list(APPEND EXCLUDED_SOURCE_PATHS_FOR_UNIT_TEST
       ${CMAKE_CURRENT_SOURCE_DIR}/jit_kernel_test.cpp
       ${CMAKE_CURRENT_SOURCE_DIR}/registers_pool.cpp
-      ${CMAKE_CURRENT_SOURCE_DIR}/ngraph_transformations/x64
+      ${CMAKE_CURRENT_SOURCE_DIR}/ov_transformations/x64
```

Comment:
may we can just keep `transformations`? without specifying whether they are `ov` or `ngraph`

Reply:
Thanks, done!